src/main/java/frc/robot/Robot.java

```diff
@@ -23,7 +33,7 @@ public void populateDashboard() {
 
   @Override
   public void setDefaultCommands() {
-    // TODO Auto-generated method stub
-
+    drive.setDefaultCommand(drive.fieldCentricDrive(() -> driveController.getLeftX(),
```

Comment:
Pro tip: you can use `driveController::getLeftX` instead of `() -> driveController.getLeftX()`

Reply:
why is this?

Reply:
Just a little shorter - you're taking a reference to the function itself, instead of making a lambda that calls it. (Theoretically it might be a little faster, but I wouldn't be bothered about that)